src/updater/updater.cpp

```diff
@@ -543,6 +694,40 @@ Updater::LoadAndUpdateEdgeExpandedGraph(std::vector<extractor::EdgeBasedEdge> &e
                        });
     }
 
+    if (SupportsShapefiles())
+    {
+        if (update_conditional_turns)
+        {
+            Timezoner time_zone_handler;
```

Comment:
Or, if it is something simple, like in this case, use `?:` operator:

    const auto time_zone_handler = config.valid_now <= 0
        ? Timezoner(config.tz_file_path)
        : Timezoner(config.tz_file_path, config.valid_now);

which is not quite as ugly as a lambda.